lib/pages/ship_fitting/bloc/ship_fitting_bloc/bloc.dart

```diff
@@ -186,13 +186,17 @@ class ShipFittingBloc extends Bloc<ShipFittingEvent, ShipFittingState> {
         },
       ).toList();
     }
-
+    // Filter out non integratable rigs
+    initialItems = initialItems.where(
```

Comment:
This `.where` clause can be chained to the one above as well :)
```suggestion
initialItems = items.where(
        (item) {
          return !integratedModifierGroups.any(
            (modifierCode) =>
                item.mainCalCode?.startsWith(modifierCode) ?? false,
          );
        },
      ).where(
            (item) => !_itemRepository.excludeFusionRigs.any((itemId) => item.id == itemId)
    ).toList();
```